datalad/distribution/add_sibling.py

```diff
@@ -0,0 +1,183 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""High-level interface for adding a sibling to a dataset
+"""
+
+__docformat__ = 'restructuredtext'
+
+
+import logging
+
+from os.path import join as opj, abspath, basename
+from datalad.support.param import Parameter
+from datalad.support.constraints import EnsureStr, EnsureNone
+from datalad.support.gitrepo import GitRepo
+from datalad.cmd import Runner
+from ..interface.base import Interface
+from datalad.distribution.dataset import EnsureDataset, Dataset, datasetmethod
+from datalad.utils import getpwd
+
+
+lgr = logging.getLogger('datalad.distribution.add_publication_target')
+
+
+class AddSibling(Interface):
+    """Adds a sibling to a dataset."""
+
+    _params_ = dict(
+        # TODO: Somehow the replacement of '_' and '-' is buggy on
+        # positional arguments
```

Comment:
This is the 2nd file with this comment. What is the issue? Shouldn't it become a proper issue?